pkg/tinydtls/patches/0001-dtls_time-migrate-to-ztimer64_usec.patch

```diff
@@ -0,0 +1,52 @@
+From 4408e8bd0c67efe6e08606568f8642f33e2fffdf Mon Sep 17 00:00:00 2001
+From: Francisco Molina <femolina@uc.cl>
+Date: Mon, 24 Jan 2022 17:54:46 +0100
+Subject: [PATCH] dtls_time: migrate to ztimer64_usec
+
+---
+ dtls_time.c | 4 ++--
+ dtls_time.h | 5 +++--
+ 2 files changed, 5 insertions(+), 4 deletions(-)
+
+diff --git a/dtls_time.c b/dtls_time.c
+index 5d353b4..4ae4414 100644
+--- a/dtls_time.c
++++ b/dtls_time.c
+@@ -43,12 +43,12 @@ dtls_tick_t dtls_clock_offset;
+ 
+ void
+ dtls_clock_init(void) {
+-  dtls_clock_offset = xtimer_now64().ticks64;
++  dtls_clock_offset = ztimer64_now(ZTIMER64_USEC);
+ }
+ 
+ void
+ dtls_ticks(dtls_tick_t *t) {
```

Comment:
in the upstream default configuration this provides msec ticks

https://github.com/eclipse/tinydtls/blob/706888256c3e03d9fcf1ec37bb1dd6499213be3c/dtls_time.h#L52

maybe the whole package can use ztimer msec

for contiki the dtls_tick_t is uint_32 

seems like there might be no need for ztimer64  
at least not for ztimer64_usec

Reply:
Do you mean `ztimer64_msec` then? Because I'm a bit worried about the rollover with 32bit msec.

Reply:
The tinydtls people are not concerned over the 32Bit msec (at least in the contiki case) i don see time beeing part of the messages of rfc4347 and all timout  are in the range of a couple of seconds -> why go 64 bit.
If tinydtls would have a problem going with 32Bit msecs this would be a bug in tinydtls which should also be fixed in interest of contiki.

https://github.com/eclipse/tinydtls/blob/706888256c3e03d9fcf1ec37bb1dd6499213be3c/dtls_time.h#L46

Reply:
Shall I squash @kfessel?

Reply:
@kfessel since https://github.com/eclipse/tinydtls/pull/123#issuecomment-1028035271 It makes me thinks in a first stage its better to leave it in ms, what about `ztimer64_msec`, as a first step, once questions in https://github.com/eclipse/tinydtls/pull/123#issuecomment-1028035271 are clarified maybe we can remove the 64bit part.

Reply:
sorry for that late answer:

we can certainly go that route i think they will sort that out and i am much less against ztimer64_msec use (than zt64us)

they also will sort out https://github.com/eclipse/tinydtls/pull/124